README.md

```diff
@@ -177,22 +177,98 @@ The webhook is conservative and allows changes whenever an error occurs:
 
 Changing the replicas number to `null` (or from `null`) is allowed.
 
+### `/admission/pre-downscale`
+
+This webhook offers a `MutatingAdmissionWebhook` that calls a downscale preparation endpoint on the ingesters for requests that decrease the number of replicas in objects labeled as `grafana.com/prep-downscale: true`.
```

Comment:
Same ingester reference comment (there is another one further down).

Reply:
Both have been changed to pod.